crates/teloxide-core/src/payloads/codegen.rs

```diff
@@ -56,10 +56,15 @@ fn codegen_payloads() {
             .map(|field| format!("    @[multipart = {}]\n", field.join(", ")))
             .unwrap_or_default();
 
+        // FIXME: CreateNewStickerSet has to be be only Debug + Clone + Serialize (maybe
+        // better fix?)
```

Comment:
I thought that such decision (putting the `CreateNewStickerSet` there) is a "dirty solution" and just a workaround

Reply:
I mean, the input file is not comparable by definition (it may include a `dyn AsyncRead` or something, which you can't compare). So you can't really do anything here.